docs/get.md

```diff
@@ -10,14 +10,13 @@ URL can be one of:
 * dataset in GitHub (e.g., https://github.com/datasets/co2-ppm)
 * direct URL to dataset
 
-## Options:
+**Options:**
```

Comment:
Why are we switching from headings to bold? Having markdown headings is nice if we ever autogenerate documentation. Please can you state why the change in the commit message.

Reply:
@rufuspollock because our markdown processor could not process headings into terminal. It will show `## Options` when you run `data get --help`.

Reply:
But ## Options is what it should show ...

OK, i'm happy to have this for now 😄

Reply:
@Mikanebu misunderstanding here is that you expected "## Some text" to be rendered as large text like it does on hackmd etc. But it is rendered differently in the terminal so "##" is still there to show it is heading level 2 (notice it has different text color). Basically, you didn't need to change this but it's OK.

You can see example here - https://www.npmjs.com/package/marked-terminal#example

Reply:
@anuveyatsu ok, in that case can we change back to having ## stuff ... ?

@anuveyatsu i'll let you call this. I'd like to get this consistently right going forward so worth getting right now.